<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PhaseTypeSelectionWidget</class>
 <widget class="QWidget" name="PhaseTypeSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>188</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item row="0" column="1">
    <widget class="QComboBox" name="dataContainerList"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Property</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QScrollArea" name="m_PhaseTypeScrollArea">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="m_PhaseTypeScrollContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>248</width>
        <height>180</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="horizontalSpacing">
        <number>6</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="attributeMatrixList"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
